fluentd/Dockerfile.centos7

```diff
@@ -25,8 +28,15 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
       libcurl-devel \
       make \
       bc \
+      bzip2 \
       iproute && \
     yum clean all
+
+RUN mkdir -p ${HOME} && \
```

Comment:
In order for us to use downstream, why not actually vendor in the dependency instead of pulling it in via curl?

Reply:
@jcantrill I don't know if we can do downstream without an rpm package . . . but sure, I can vendor it in - I just didn't want to check in a big tarball into git.  I suppose I could check in the uncompressed source tree, but if I'm going to do that, I should just make it a git submodule

Reply:
I would prefer not to vendor the source but if that is our only recourse we should do it.  Alternatively, maybe its worth us setting up the necessary repos to properly build it into an rpm.  Is that something you are willing to take on?

Reply:
@jcantrill for downstream. separate rpm for the rhlog channel, then we just tag those builds in to build the fluentd image